sandbox/settings.py

```diff
@@ -367,6 +367,7 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
                     "richie.apps.core.context_processors.site_metas",
                     "social_django.context_processors.backends",
                     "social_django.context_processors.login_redirect",
+                    "richie.apps.core.context_processors.google_analytics_cp",
```

Comment:
Should we add it to the existing Richie context processor in order not to complicate configuration?

Reply:
Ok.